scripts/release.ts

```diff
@@ -21,8 +21,18 @@ export async function release(rootDir: string, args: ReadonlyArray<string>): Pro
       isPublishRelease: false,
       isProd: true,
     });
-    const answers = await promptPrepareRelease(opts);
-    return prepareRelease(opts, args, releaseDataPath, answers);
+
+    const responses = await promptPrepareRelease(opts);
+    opts.version = responses.versionToUse;
+
+    if (!responses.confirm) {
+      console.log(`\n${color.bold.magenta('Publish preparation cancelled by user')}\n`);
+      return;
+    }
```

Comment:
This has been hoisted out of [prepareRelease](https://github.com/ionic-team/stencil/pull/4235/files#diff-21f9896491d70e481e031a1c66e9ffed07592f08455bdcd4a3cecc9c63814768L50).